include/cpuinfo_x86.h

```diff
@@ -167,6 +167,7 @@ typedef enum {
   AMD_EXCAVATOR,       // K15 EXCAVATOR
   AMD_BULLDOZER,       // K15 BULLDOZER
   AMD_JAGUAR,          // K16 JAGUAR
+  AMD_CATO,            // K16 CATO
```

Comment:
I see that in instant site mentions Jaguar: AMD A9-9820 (Cato, 2x4c Jaguar core, RDRAND disabled). I didn't find any info about uarch Cato, only the code name "Cato", so I guess it should be AMD_JAGUAR

Reply:
ok